Unit-01/02-flask-routing/app.py

```diff
@@ -0,0 +1,22 @@
+# from the flask library import a class named Flask
+from flask import Flask
+import math
+app = Flask(__name__)
+
+
+@app.route('/math/<int:num>/<string:op>/<int:num2>')
+def nums(num, op, num2):
+    if op == '+':
+        return f"{num + num2}"
+    if op == '-':
+        return f"{num - num2}"
+    if op == '*':
+        return f"{num * num2}" 
+    if op == '%':
+        return f"{math.floor(num / num2)}"
+# I tried assigning the math operators to a dictionary,
```

Comment:
Hey @CJTiago - let's chat about this. Using a dictionary should be fine. Do you have an old version of the code you can share?